portal-frontend/src/app/features/edit-submission/primary-contact/primary-contact.component.ts

```diff
@@ -135,7 +135,11 @@ export class PrimaryContactComponent extends FilesStepComponent implements OnIni
         this.isCrownOwner = selectedOwner.type.code === APPLICATION_OWNER.CROWN;
       }
     }
-    this.needsAuthorizationLetter = !selectedOwner;
+    this.needsAuthorizationLetter = !(
+      this.owners[0].type.code === APPLICATION_OWNER.INDIVIDUAL &&
+      (this.owners.length === 1 ||
+        (this.owners.length === 2 && this.owners[1].type.code === APPLICATION_OWNER.AGENT && !hasSelectedAgent))
```

Comment:
What does this second check cover for? I feel like the original rule of owner.length == 1 and owner is individual is the ONLY case where we don't require an auth letter.